project/dexter-metrics/src/test/com/samsung/sec/dexter/metrics/util/TestFile_For_MetricUtilTest.java

```diff
@@ -0,0 +1,23 @@
+package com.samsung.sec.dexter.metrics.util;
+
+import javax.swing.text.StyledEditorKit.ForegroundAction;
+
+public class TestFile1_For_MetricUtilTest {
```

Comment:
build error occurs in Eclipse.
The class name is not the same as the file name.